catroid/src/main/res/layout/brick_tap_for.xml

```diff
@@ -0,0 +1,88 @@
+  ~ it under the terms of the GNU Affero General Public License as
+  ~ published by the Free Software Foundation, either version 3 of the
+  ~ License, or (at your option) any later version.
+  ~
+  ~ An additional term exception under section 7 of the GNU Affero
+  ~ General Public License, version 3, is available at
+  ~ http://developer.catrobat.org/license_additional_term
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_vertical"
+    android:orientation="horizontal">
+
+    <CheckBox
+        android:id="@+id/brick_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        android:visibility="gone" />
+
+    <org.catrobat.catroid.ui.BrickLayout
+        android:id="@+id/brick_place_at_layout"
+        style="@style/BrickContainer.Test.Medium">
+
+        <TextView
+            style="@style/BrickText.SingleLine"
+            android:text="@string/brick_tap_at" />
+        
+        <TextView
+            style="@style/BrickText.SingleLine"
+            android:text="@string/x_label"
+            app:layout_newLine="true" />
+
+        <TextView
+            android:id="@+id/brick_tap_for_edit_x"
+            style="@style/BrickEditText"
+            app:layout_inputType="number"
+            app:layout_textField="true" />
+
+        <TextView
+            style="@style/BrickText.SingleLine"
+            android:paddingStart="@dimen/material_design_spacing_small"
+            android:text="@string/y_label"
+            android:paddingEnd="@dimen/material_design_spacing_small" />
+
+        <TextView
+            android:id="@+id/brick_tap_for_edit_y"
+            style="@style/BrickEditText"
+            app:layout_inputType="number"
+            app:layout_textField="true" />
+
+        <TextView
+            style="@style/BrickText.SingleLine"
+            android:text="@string/brick_tap_for"
+            app:layout_newLine="true" />
+
+        <TextView
+            android:id="@+id/brick_tap_for_edit_duration"
+            style="@style/BrickEditText"
+            app:layout_inputType="number"
+            app:layout_textField="true" />
+
+        <TextView
+            android:id="@+id/brick_seconds_label"
+            style="@style/BrickText.SingleLine" />
+
+        <include layout="@layout/icon_brick_category_look" />
```

Comment:
the icon layout must be the first childView in the BrickLayout.
please move this `<include layout="@layout/icon_brick_category_look" />` to line 42 before TextViews